app/uk/gov/hmrc/servicedependencies/service/SlugDependenciesService.scala

```diff
@@ -39,24 +39,21 @@ class SlugDependenciesService @Inject() (slugInfoRepository: SlugInfoRepository,
   def curatedLibrariesOfSlug(name: String, version: String): Future[Option[List[Dependency]]] = {
     val futLatestVersionByName = libraryVersionRepository.getAllEntries.map(toLatestVersionByName)
```

Comment:
Is it intentional that these futures are launched in parallel? If not, you could just inline them into the for-comprehension.
If it is, using cats `.mapN` can make this explicit.

Reply:
Yes, that is intentional, as each of these futures represents an independent source of 'input' to the calculation.  If this was to be in-lined I doubt that the performance implications would be significant - but it seemed sensible to keep things parallel when there was little complexity in doing so.

As the function was so simple I didn't feel it warranted reaching for cats, but am happy to change if that is the preferred approach.